tests/integrationv2/providers.py

```diff
@@ -807,3 +897,30 @@ def supports_cipher(cls, cipher, with_curve=None):
     @classmethod
     def supports_signature(cls, signature):
         return GnuTLS.sigalg_to_priority_str(signature) is not None
+
+
+
+def find_files(file_glob, root_dir=".", modes=[]):
```

Comment:
Nit. Did you run a formatter on this? Does our python formatter not check whitespace?
```suggestion
        return GnuTLS.sigalg_to_priority_str(signature) is not None


def find_files(file_glob, root_dir=".", modes=[]):
```

Reply:
Long story- after rewritting the .pep8 file a few times, it's in a place were it's not being too aggressive (for this PR).